### PR TITLE
Add click-completion

### PR DESCRIPTION
**Pull request by epruesse**

### CODE REVIEW

File: recipes/click-completion/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/d8/10/e2eead22c1893c2bd717b4dee55c72de5793fce38ee231211dd3858b012a/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (CJ-Wright) [inline recipes/click-completion/meta.yaml]: Could this be noarch?

File: recipes/click-completion/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (CJ-Wright) [inline recipes/click-completion/meta.yaml]: could this use pip https://conda-forge.org/docs/meta.html#use-pip

File: recipes/click-completion/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - click
+    - jinja2
```
Comment (CJ-Wright) [inline recipes/click-completion/meta.yaml]: are these all build requirements?

File: recipes/click-completion/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
+  imports:
+    - click_completion
+
+about:
+  home: https://github.com/click-contrib/click-completion
+  license: MIT
```
Comment (CJ-Wright) [inline recipes/click-completion/meta.yaml]: Is there a license_file?

File: recipes/click-completion/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+{% set version = "0.2.1" %}
+{% set sha256 = "079fb138887d4de12a0b7fbebf8d92d396b7c1a9c49f63475d9f3909d2588976" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/d8/10/e2eead22c1893c2bd717b4dee55c72de5793fce38ee231211dd3858b012a/{{ name }}-{{ version }}.tar.gz
```
Comment (CJ-Wright) [inline recipes/click-completion/meta.yaml]: Does a url of this form work?
`https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`

File: recipes/click-completion/meta.yaml (hunk 4 above)
Comment (epruesse) [reply recipes/click-completion/meta.yaml]: Yes, but it's omitted from the release archives. 
https://github.com/click-contrib/click-completion/blob/master/LICENSE

File: recipes/click-completion/meta.yaml (hunk 2 above)
Comment (epruesse) [reply recipes/click-completion/meta.yaml]: Fixed

File: recipes/click-completion/meta.yaml (hunk 4 above)
Comment (CJ-Wright) [reply recipes/click-completion/meta.yaml]: Would you mind asking the code maintainers to add it to the package (and put a link to the issue in the meta.yaml as a comment)?

File: recipes/click-completion/meta.yaml (hunk 3 above)
Comment (epruesse) [reply recipes/click-completion/meta.yaml]: fixed

File: recipes/click-completion/meta.yaml (hunk 5 above)
Comment (epruesse) [reply recipes/click-completion/meta.yaml]: yes; fixed

File: recipes/click-completion/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (CJ-Wright) [inline recipes/click-completion/meta.yaml]: could this be noarch?

File: recipes/click-completion/meta.yaml (hunk 4 above)
Comment (epruesse) [reply recipes/click-completion/meta.yaml]: Just pushed a PR

File: recipes/click-completion/meta.yaml (hunk 1 above)
Comment (epruesse) [reply recipes/click-completion/meta.yaml]: fixed

File: recipes/click-completion/meta.yaml (hunk 6 above)
Comment (epruesse) [reply recipes/click-completion/meta.yaml]: fixed

**Comment by epruesse** [comment]
@CJ-Wright I can create a `build.sh` to fix the license issue if necessary:

```
pip install....
wget https://raw.githubusercontent.com/click-contrib/click-completion/0.2.1/LICENSE
``` 

should do it

**Comment by CJ-Wright** [comment]
If you want to fix the licence I'd do [this](https://conda-forge.org/docs/meta.html#packaging-the-license-manually)

**Comment by epruesse** [comment]
Well, the MIT license does not require redistribution with license. So I think we're good the way it is?

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes thoughts on the licence?

**Comment by isuruf** [comment]
https://github.com/click-contrib/click-completion/blob/master/LICENSE#L12-L13

"The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software."

**Comment by CJ-Wright** [comment]
Should we have something in the linter which knows which license families require their licenses to be in the package?

**Comment by epruesse** [comment]
@isuruf Darn. I somehow missed that. Thanks!

**Comment by sodre** [comment]
@conda-forge/staged-recipes can someone check if this PR can be merged? I am happy to be added as a co-maintainer in case there are additional changes needed.

**Pull request merged**